src/Main.elm

```diff
@@ -181,13 +191,24 @@ view model =
                     text ""
                 , renderGameArea model
                 , renderHelp model
+                , div [] (model.playedSoundEffects |> List.map renderAudio)
                 ]
             ]
         , footer [ class "page-section" ]
             [ a [ href "https://wearedarts.org.uk" ] [ text "wearedarts.org.uk" ] ]
         ]
 
 
+renderAudio : SoundEffect -> Html Msg
+renderAudio soundEffect =
+    Html.audio
```

Comment:
I've used the Html namespace here since they are not standard components - to make it obvious which library they come from.

Reply:
Looks like just a `<audio>` element rather than anything from a library?

Reply:
Yeah there is no library apart from Elm core `Html` involved. It is just an `<audio>` element. I thought if I put it in as `audio` there might be some confusion as to whether is was just Html